app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/SettingsViewModel.kt

```diff
@@ -97,13 +80,18 @@ class SettingsViewModel @Inject constructor(
         try {
             activity.startActivity(goToMarket)
         } catch (e: ActivityNotFoundException) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/636d317b685d72b06abf1b59ef55421cd7244cef/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/settings/SettingsViewModel.kt#L82"></span>The caught exception is swallowed. The original exception could be lost.</td>
    </tr>
  </tbody>
</table>